src/app/lib/utilities/preprocessor/rules/filterContentType.js

```diff
@@ -1,7 +1,16 @@
 import pathOr from 'ramda/src/pathOr';
 
 const whitelist = ['STY', 'MAP', 'PGL', 'LIV', 'PRO'];
-const contentTypes = ['Text', 'Feature', 'Audio', 'Video', 'Gallery', 'Guide'];
+const contentTypes = [
```

Comment:
Since we are adding 2 new `contentTypes`, there should probably be tests for them in `filterContentType.test.js` for consistency since we have tests for all of the other `contentTypes`